doc/Protocols/SecurityServerExtension/xroad-securityserver.xsd

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema elementFormDefault="qualified"
+           targetNamespace="http://x-road.eu/xsd/xroad.xsd"
+           xmlns="http://x-road.eu/xsd/xroad.xsd"
+           xmlns:id="http://x-road.eu/xsd/identifiers"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:import namespace="http://www.w3.org/XML/1998/namespace"
+             schemaLocation="http://www.w3.org/2001/xml.xsd"/>
```

Comment:
http://x-road.eu/xsd/xroad.xsd is using http://www.w3.org/2009/01/xml.xsd schema location, and to avoid validation problems extension must use the same.